.travis.yml

```diff
@@ -77,6 +77,10 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  #
+  # Some change in virtualenv 14.0.5 caused `test_f2py` to fail. So, we have
+  # pinned `virtualenv` to the last known working version to avoid this failure.
```

Comment:
Haha, yeah, would be nice. Figured I'd just end the stream of Travis failed emails instead.

Honestly, it is cases like this where I start preferring a different sort of CI all together like CircleCI where one can inspect the VM or Wercker where one can not only inspect a failed build, but can run the CI locally. The hack stuff into the CI script and wait a few minutes before more hacking just doesn't cut it for me.

Yep, I did discover that recently. Still this somehow fixes it. Kind of a mystery why atm.